src/components/Select/Select.js

```diff
@@ -34,13 +35,12 @@ const Select = ({
       label={label}
       required={required}
       stacked={stacked}
-      success={success}
-    >
+      success={success}>
       <select
         className={classNames("p-form-validation__input", className)}
         id={id}
-        {...props}
-      >
+        onChange={evt => onChange(evt.target.value)}
```

Comment:
can we just pass the evt through. also the onChange is optional, so we should only call it if it's defined.

```suggestion
        onChange={evt => onChange && onChange(evt)}
```